doc/reference/reference_lua/config.rst

```diff
@@ -351,178 +380,355 @@ config API
 
-.. module:: config.storage
+.. _config_storage_client_api_reference:
 
-.. _config_storage_api_reference_put:
+.. class:: config.storage_client
 
-.. function:: put(path, value)
+The ``config.storage_client`` API allows you to connect to a remote replica set :ref:`configured as a Tarantool-based centralized configuration storage <configuration_etcd>` and interact with it as a key-value storage, see Client API examples below.
 
-    Put a value by the specified path.
+.. _config_storage_client_api_reference_connect:
 
-    :param string path: a path to put the value by
-    :param string value: a value to put
+.. function:: connect(endpoints[, options])
 
-    :return:    a table containing the following fields:
+    Connect to a remote config.storage cluster.
 
-                *   ``revision``: a revision after performing the operation
+    :param table endpoints: table of config.storage instance URIs
+    :param table options: ``timeout`` - number, connect & call timeout (default: ``nil``, unset)
+
+    :return:    a <config.storage client> object
 
     :rtype: table
 
-    **Example:**
+The example below shows how to connect to a specific config.storage cluster:
 
-    The example below shows how to read a configuration stored in the ``source.yaml`` file using the :ref:`fio module <fio-module>` API and put this configuration by the ``/myapp/config/all`` path:
+..  literalinclude:: /code_snippets/snippets/centralized_config/instances.enabled/config_storage/myapp.lua
+    :language: lua
+    :start-after: config = require
+    :end-at: })
+    :dedent:
 
-    ..  literalinclude:: /code_snippets/snippets/centralized_config/instances.enabled/tarantool_config_storage/myapp.lua
-        :language: lua
-        :start-after: function put_config
-        :end-at: cluster_config_handle:close()
-        :dedent:
+The example below shows how to connect to a config storage cluster configured as the :ref:`centralized configuration storage <configuration_etcd>` in the :ref:`config.storage <configuration_reference_config_storage>` option:
 
-    Example on GitHub: `tarantool_config_storage <https://github.com/tarantool/doc/tree/latest/doc/code_snippets/snippets/centralized_config/instances.enabled/tarantool_config_storage>`_
+..  literalinclude:: /code_snippets/snippets/centralized_config/instances.enabled/config_storage/myapp.lua
+    :language: lua
+    :start-after: function connect_to_configured_storage
+    :end-at: config.storage_client.connect
+    :dedent:
 
+Examples on GitHub: `config_storage <https://github.com/tarantool/doc/tree/latest/doc/code_snippets/snippets/centralized_config/instances.enabled/config_storage>`_
 
-.. _config_storage_api_reference_get:
+.. _config_storage_object_api_reference_put:
 
-.. function:: get(path)
+.. function:: <config.storage server>.put(path, value)
+.. function:: <config.storage client>:put(path, value)
```

Comment:
How about the `ttl` option? https://github.com/tarantool/enterprise_doc/issues/281